cmd/activator/main.go

```diff
@@ -246,22 +246,12 @@ func main() {
 	cr := activatorhandler.NewConcurrencyReporter(podName, reqChan, time.NewTicker(time.Second).C, statChan)
 	go cr.Run(stopCh)
 
-	ah := &activatorhandler.ProbeHandler{
-		NextHandler: &activatorhandler.FilteringHandler{
-			NextHandler: activatorhandler.NewRequestEventHandler(reqChan,
-				&activatorhandler.EnforceMaxContentLengthHandler{
-					MaxContentLengthBytes: maxUploadBytes,
-					NextHandler: &activatorhandler.ActivationHandler{
-						Activator: a,
-						Transport: rt,
-						Logger:    logger,
-						Reporter:  reporter,
-						Throttler: throttler,
-					},
-				},
-			),
-		},
-	}
+	var ah http.Handler
+	ah = &activatorhandler.ActivationHandler{Activator: a, Transport: rt, Logger: logger, Reporter: reporter, Throttler: throttler}
+	ah = &activatorhandler.EnforceMaxContentLengthHandler{MaxContentLengthBytes: maxUploadBytes, NextHandler: ah}
+	ah = activatorhandler.NewRequestEventHandler(reqChan, ah)
+	ah = &activatorhandler.FilteringHandler{NextHandler: ah}
+	ah = &activatorhandler.ProbeHandler{ah}
```

Comment:
Please add a comment that these will be executed inside out. The nesting made the order of handlers quite clear, now it'll be backwards.

Reply:
Cool, added